### ISSUE TITLE
Add default support for most common media formats. 

### ISSUE THREAD

**Issue by shapeh** [PiranhaCMS/piranha.core#1359]
According to the docs - https://piranhacms.org/docs/content/media

The following are supported by default but perhaps it is time to include .gif, .webp and .jfif as common formats too (by default):

> .jpg, .jpeg & .png images
> .mp4 videos
> .pdf documents

https://github.com/PiranhaCMS/piranha.core/issues/237 

We can do this:

Piranha.App.MediaTypes.Images.Add(".gif", "image/gif");
Piranha.App.MediaTypes.Images.Add(".jfif", "image/jpeg");
...

https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types

**Comment by tidyui** [comment]
The reason we support the specified formats is that need to consider which formats are supported by `ImageSharp`, which is the library we currently use for scaling and cropping images.

https://docs.sixlabors.com/articles/imagesharp/imageformats.html

It appears by the documentation that they now supports `.gif` but we have deliberately stayed clear of `.gif` as we're not sure what will happen if one would try to attempt scaling operations on an animated gif. Still if someone would have time investigating this we could very well include it in the default file types. Currently `.webp` and `.jfif` seems to be unsupported.

**Comment by shapeh** [comment]
Thanks for a quick reply:

I threw together a quick method to get all supported formats:
```
using SixLabors.ImageSharp;
using SixLabors.ImageSharp.Formats;
using SixLabors.ImageSharp.Processing;
using System;
```
        private static void GetFormats()
        {
            foreach (IImageFormat F in Configuration.Default.ImageFormats)
            {
                Console.Write($"  {F}  Mime Type: {F.DefaultMimeType}  Extension(s):  ");
                foreach (string ext in F.FileExtensions)
                {
                    Console.Write($"{ext}  ");
                }
                Console.WriteLine();
            }
        }

```
  SixLabors.ImageSharp.Formats.Png.PngFormat  Mime Type: image/png  Extension(s):  png
  SixLabors.ImageSharp.Formats.Jpeg.JpegFormat  Mime Type: image/jpeg  Extension(s):  jpg  jpeg  jfif
  SixLabors.ImageSharp.Formats.Gif.GifFormat  Mime Type: image/gif  Extension(s):  gif
  SixLabors.ImageSharp.Formats.Bmp.BmpFormat  Mime Type: image/bmp  Extension(s):  bm  bmp  dip
  SixLabors.ImageSharp.Formats.Tga.TgaFormat  Mime Type: image/tga  Extension(s):  tga  vda  icb  vst
```
jfif, jpeg and jpg are all the same. If you look at the header of a jpg/jpeg/jfif file you will see JFIF.
As for .gif - it also works fine with resizing (although coloring in this example seems to be a bit off - could probably be corrected):


        private static void Resize()
        {
            using (Image image = Image.Load("animated.gif"))
            {
                image.Mutate(x => x.Resize(image.Width / 2, image.Height / 2));
                image.Save("resized.gif");
            }
        }

![animated](https://user-images.githubusercontent.com/29387615/95720128-1bab9700-0c71-11eb-8cb8-6c25f84b3827.gif)
![resized](https://user-images.githubusercontent.com/29387615/95720143-1f3f1e00-0c71-11eb-92a3-bc87c774063e.gif)

**Comment by magnuskarlssonhm** [comment]
Hey! Imagesharp supports webp now :) Reopen? 
https://docs.sixlabors.com/articles/imagesharp/imageformats.html

**Comment by tidyui** [comment]
@magnuskarlssonhm I think perhaps we should open a new issue for .webp support as this issue was released in 9.1
